src/main.rs

```diff
@@ -286,9 +325,28 @@ fn run() -> Result<(), failure::Error> {
                 let title = rename_matches.value_of("title").unwrap();
                 commands::kv::rename_namespace(id, title)?;
             }
-            ("list", Some(_create_matches)) => {
+            ("list", Some(_list_matches)) => {
                 commands::kv::list_namespaces()?;
             }
+            ("write", Some(write_matches)) => {
+                match write_matches.subcommand() {
+                    ("bulk", Some(bulk_write_matches)) => {
+                        let id = bulk_write_matches.value_of("id").unwrap();
+                        let filename = bulk_write_matches.value_of("filename").unwrap();
+                        let expiration = bulk_write_matches.value_of("expiration");
+                        let ttl = bulk_write_matches.value_of("time-to-live");
+                        let base64 = match bulk_write_matches.occurrences_of("release") {
+                            1 => true,
+                            _ => false,
+                        };
+                        commands::kv::write_bulk(id, filename, expiration, ttl, base64)?;
```

Comment:
expiration, ttl, and base64 are actually all set on individual objects in the JSON. We may want to provide flags for setting them for the whole set (especially for uploading a whole directory), but we'll have to decide preference order.

Reply:
I think it's worth keeping the consistency of this API usage by ensuring that folks using it know that these options apply to all things uploaded in a bulk upload action--if folks want to do this for individual key-value pairs (or files for static site hosting) they can use the individual `write` subcommand?

Reply:
Hmm, now that I think about it, I'm okay with forcing folks to specify expiration and ttl in their json (adding them as commandline flags would imho get clunky and confusing to use for bulk). What do you think? (This is following our convo earlier today :)). 

We can still provide these as flags for individual writs!